src/test/java/com/github/netelli/ProductsDAOTest.java

```diff
@@ -32,8 +32,7 @@ public void testInsertData() throws Exception {
         List<Product> products = productsDAO.getProducts();
 
         assertEquals(3, products.size());
-        //TODO fix test
-//        assertEquals("skirt mini", products.get(0));
+        assertEquals("skirt mini", products.get(0).getTitle());
```

Comment:
It may cause NullPointerException, at first assert that products.get(0) is not null by using assertNotNull(...) and than do what you do.